### ISSUE TITLE
Selecting a tab in Meta-Example may not be applied immediately.

### ISSUE THREAD

**Issue by kojix2** [AndyObtiva/glimmer-dsl-libui#8]
Today I tried meta_xeample.rb with glimmer-dsl-libui.

I found a very small unexpected behavior. 

After launching `meta_example.rb`, open the Advanced tab ①,  and you will see "Area Gallery (4 versions)" selected. 
Now, Click on the Launch button ②. The "Basic Area" will be launched. This was an unexpected behavior for me.

For practical use, there is no problem at all, but I would like to report it.
Have a good day.

![image](https://user-images.githubusercontent.com/5798442/145698219-c13609ca-ab1c-474b-b267-27c3cc1284ef.png)

* Ubuntu 21.10 impish
* ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux]
* glimmer-dsl-libui (0.4.18)

**Comment by AndyObtiva** [comment]
Yes, I noticed this behavior on Linux only. On the Mac and Windows, the selected example under Basic remains selected when I switch to the Advanced tab, so it is not confusing to the user.

![Screen Shot 2021-12-11 at 11 59 55 PM](https://user-images.githubusercontent.com/23052/145700997-b19ba253-ee94-4ca9-bce2-ea9e56145472.png)

![glimmer-dsl-libui-windows-meta-example-advanced](https://user-images.githubusercontent.com/23052/145701001-79c35c0f-7827-4534-b17a-be7fa6278804.png)

Thank you for reporting. Unfortunately, it is not a Glimmer DSL for LibUI issue, yet a [C libui](https://github.com/andlabs/libui) issue. Perhaps, you could report there instead and link back here just for reference.

**Comment by kojix2** [comment]
Thanks.

I understand your intended behavior on macOS and Windows.
The expected behavior is that the radio buttons are not selected when a tab So it is a problem with c libui that the radio buttons are selected from the beginning. Also, as you said, I have confirmed that there is no function to detect that the tab has been switched.

I thought about checking the tab number when the start button is pressed, but that would be useless if the code displayed in the text area is not updated when the tab is switched. I see, it is not an easy fix.

By the way, you might be wondering why I'm running meta_example today. Actually, I wrote a small [post](https://qiita.com/kojix2/items/47011057d3e9c730d6da) on Qiita to introduce glimmer-dsl-libui to Rubyists in Japan. This is a website equivalent to dev.to in Japanese. I hope it will help more people in Japan to discover glimmer-dsl-libui.

Take care again.
